code/modules/mob/living/carbon/xenomorph/castes/hivelord/abilities_hivelord.dm

```diff
@@ -1,3 +1,11 @@
+//Some debug variables. Toggle them to 1 in order to see the related debug messages. Helpful when testing out formulas.
+#define DEBUG_HIVELORD_ABILITIES	0
+#if DEBUG_HIVELORD_ABILITIES
```

Comment:
Ifdef

Reply:
This actually doesn't work unfortunately; I tried it and had to revert, unless you're telling me to ifdef all of the debug message instances which is a less prefable solution.

Reply:
No I mean just
//define DEBUG_HIVELORD_ABILITIES "ae"
#ifdef DEBUG_HIVELORD_ABILITIES

Reply:
pending still